public/video-ui/src/components/Atoms/AtomItem.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {Link} from 'react-router';
+
+export default class AtomItem extends React.Component {
+
+  render() {
+    return(
+        <li className="grid__item">
```

Comment:
I just made it a bit more generic as it's kind of a grid view. But as we're only using it for videos/media atoms here, it makes sense to be a bit more specific